app/services/data_migrations/update_provider_user_send_notifications.rb

```diff
@@ -0,0 +1,16 @@
+module DataMigrations
+  class UpdateProviderUserSendNotifications
+    TIMESTAMP = 20210315142623
+    MANUAL_RUN = false
+
+    def change
+      ProviderUserNotificationPreferences.find_each do |preferences|
+        send_notifications = ProviderUserNotificationPreferences::NOTIFICATION_PREFERENCES.map { |pref| preferences.send(pref) }.any?(true)
+
+        if preferences.provider_user.send_notifications != send_notifications
```

Comment:
I think there is a misunderstanding here, as this should be upserting the `ProviderUserNotificationPreferences` similar to the other PR, rather than setting `send_notifications`? 
The current implementation is best to happen in Aga's controller every time someone goes through the new flow as we can't run this migration more than once but have to maintain consistency.

Happy to jump on a call to clarify